global.json

```diff
@@ -1,6 +1,9 @@
 {
+  "sdk": {
+    "version": "6.0.300-preview"
+  },
   "tools": {
-    "dotnet": "6.0.100-rc.2.21478.25"
+    "dotnet": "6.0.300-preview"
```

Comment:
This was an entry that dotnet/arcade tooling uses on our internal CI. They have powershell scripts and things that read this number. I think you have to put the full version number to change this, it doesn't know how to roll forward.

Can we just put the latest stable .NET release? Or not change this?

Reply:
Hmm looks like this is required to get the SDK change to work. 

https://dev.azure.com/xamarin/public/_build/results?buildId=58378&view=logs&jobId=2751d22a-42ce-5ac2-f40c-7f95ad2ff1fb&j=2751d22a-42ce-5ac2-f40c-7f95ad2ff1fb&t=70ac8f3a-d989-5549-c519-2cd4c5147626

Reply:
Nop, we should use the latest stable. this is only used like @jonathanpeppers said for CI. I can also try to remove this need if that's a issue. Should we fix the script @jonathanpeppers  ? maybe hardcode this value on the script?